### ISSUE TITLE
ts-node: not found

### ISSUE THREAD

**Issue by iLikeSalads** [phamleduy04/texas-dps-scheduler#158]
Hi All,

First of all, thank you so much for making our lives better by developing this automated application.

The issue i face is when i'm trying to start application by running `npm run start` it throws below error
`sh: 1: ts-node: not found`

i did install ts-node by doing `npm install -g ts-node` but now it throws different error like below

```
 Error: Cannot find module 'tslib'
Require stack:
- /workspaces/texas-dps-scheduler/src/index.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/usr/local/share/nvm/versions/node/v20.15.1/lib/node_modules/ts-node/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:986:27)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/workspaces/texas-dps-scheduler/src/index.ts:3:17)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module.m._compile (/usr/local/share/nvm/versions/node/v20.15.1/lib/node_modules/ts-node/src/index.ts:1618:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Object.require.extensions.<computed> [as .ts] (/usr/local/share/nvm/versions/node/v20.15.1/lib/node_modules/ts-node/src/index.ts:1621:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/workspaces/texas-dps-scheduler/src/index.ts' ]
}
```

Thank you everyone.

**Comment by dmz94** [comment]
I am also seeing this...

**Comment by Alecmay101** [comment]
doing npm install first resolved this issue for me on 4.1.0

**Comment by dmz94** [comment]
npm install fixed it for me. Pilot error on my part - THANKS!
